python/tvm/relay/op/nn/_nn.py

```diff
@@ -142,18 +153,19 @@ def schedule_conv2d(attrs, outs, target):
             return topi.generic.schedule_conv2d_nhwc(outs)
         if groups != 1:
             # collect in_channels to distinguish depthwise and group conv2d
-            wkl = outs[0].op.attrs['workload']
-            in_channels = wkl[1][1]
-            out_channels = outs[0].shape[1]
-
-            if layout == "NCHW" and in_channels == groups and in_channels == out_channels:
-                # TODO(leyuan, merrymercy, Huyuwei): fold depthwise topi into conv2d.
-                return topi.generic.schedule_depthwise_conv2d_nchw(outs)
-            if layout == "NHWC" and kernel_layout == "HWOI" and \
-                in_channels == groups and in_channels == out_channels:
-                return topi.generic.schedule_depthwise_conv2d_nhwc(outs)
-            if layout == "NCHW":
-                return topi.generic.schedule_group_conv2d_nchw(outs)
+            op = _find_conv2d_op(outs[0].op)
+            assert op is not None
+
+            is_depthwise = 'depthwise' in op.tag
```

Comment:
I'm not sure whether it is a good way to go but it seems that checking whether `depthwise` is within the `tag` value is much easier and intuitive.